### PR TITLE
MajoranaNeutrinoToTriLeptons gridpack cards added

### PR DESCRIPTION
**Pull request by jedori0228**
MajoranaNeutrinoToTriLeptons gridpack cards added.

https://docs.google.com/spreadsheets/d/1nBAko5DgS9dMslxqQcQOs2TYcpWkxmuQ2VBg5BsphBU/edit#gid=0

### CODE REVIEW

**Comment by perrozzi** [comment]
(just in case, the google doc is not accessible)

**Comment by jedori0228** [comment]
https://docs.google.com/spreadsheets/d/1nBAko5DgS9dMslxqQcQOs2TYcpWkxmuQ2VBg5BsphBU/edit?usp=sharing

**Comment by jedori0228** [comment]
Here I put shared (editable) link :

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/MajoranaNeutrinoToTriLeptons/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model sm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/MajoranaNeutrinoToTriLeptons/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO_proc_card.dat]: I suggest to remove the lines between "import model sm" and "define vl~ = ve~ vm~ vt~" for better clarity

Comment (jedori0228) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/MajoranaNeutrinoToTriLeptons/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO_proc_card.dat]: I will PR again, with those lines removed.

Comment (jedori0228) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/MajoranaNeutrinoToTriLeptons/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO_proc_card.dat]: https://github.com/cms-sw/genproductions/pull/1037

**Pull request closed without merging**